app/models/request.rb

```diff
@@ -63,23 +65,25 @@ def validator_for(request_option)
           'INNER JOIN assets AS pw ON well_links.target_well_id=pw.id AND well_links.type="stock"',
         ]
       end
-    {
-      :select => 'uuids.external_id AS pool_id, GROUP_CONCAT(DISTINCT pw_location.description ORDER BY pw.map_id ASC SEPARATOR ",") AS pool_into, requests.*',
-      :joins => joins + [
+
+
+    select('uuids.external_id AS pool_id, GROUP_CONCAT(DISTINCT pw_location.description ORDER BY pw.map_id ASC SEPARATOR ",") AS pool_into, MIN(requests.id) AS id, MIN(requests.sti_type) AS sti_type, MIN(requests.submission_id) AS submission_id, MIN(requests.request_type_id) AS request_type_id').
```

Comment:
If the select is filtering to select the requests for the stock plate, what does it mean "MIN(requests.sti_type) AS sti_type"? Is it picking up the first name of the classes for the requests but in alphabetical order?